app.js

```diff
@@ -0,0 +1,40 @@
+;(function() {
+  var list = document.querySelector("[data-js=todoItems]");
+  var form = document.querySelector("[data-js=form]");
+  var hide = document.querySelector("[data-js=hide]");
```

Comment:
calling this element hide in the markup doesn't make too much sense to me. I'd perhaps call it something else, given it's not always hidden